VerifyReplication/storage/s3.js

```diff
@@ -140,14 +127,19 @@ function listObjects(params, cb) {
         return cb(new Error(errMsg));
     }
 
-    // TODO: support listing all versions
-    return client.listObjectsV2({
-        Bucket: bucket,
-        MaxKeys: listingLimit,
-        Prefix: prefix,
-        Delimiter: delimiter,
-        ContinuationToken: nextContinuationToken,
-    }, cb);
+    try {
+        // TODO: support listing all versions
+        const data = await client.send(new ListObjectsV2Command({
+            Bucket: bucket,
+            MaxKeys: listingLimit,
+            Prefix: prefix,
+            Delimiter: delimiter,
+            ContinuationToken: nextContinuationToken,
+        }));
+        return cb(null, data);
```

Comment:
same here: catch must not wrap the callback: otherwise, if the callback throws, we will call it again....